internal/resource/config/oauth/oauth_auth_server_settings_resource.go

```diff
@@ -608,17 +609,12 @@ func (r *oauthAuthServerSettingsResource) ValidateConfig(ctx context.Context, re
 	if internaltypes.IsDefined(model.AllowedOrigins) {
 		aoElems := model.AllowedOrigins.Elements()
 		for _, aoElem := range aoElems {
-			aoElemString := aoElem.(basetypes.StringValue).ValueString()
-			isElemUrl := internaltypes.IsUrlFormat(aoElemString)
-			if !isElemUrl {
-				resp.Diagnostics.AddError("Invalid URL Format!", fmt.Sprintf("Please provide a valid origin. Origin \"%s\" needs to be in a valid URL-like format - \"http(s)//:<value>.<domain>\"", aoElemString))
-			}
+			validators.IsUrlFormat(aoElem, resp)
```

Comment:
We could implement this as a plugin framework validator so that we could include it in the schema, since this isn't dependent on any other attributes. This way works fine as well though.

Reply:
Was it not possible to move this to the schema validator?

Reply:
the recent commit has a set schema validator for this